src/stores/index.ts

```diff
@@ -0,0 +1,41 @@
+import { configureStore, createListenerMiddleware } from '@reduxjs/toolkit';
+import { CHAT_ID_STORE_KEY, CHAT_MESSAGES_STORE_KEY } from 'src/constants';
```

Comment:
WDYT about adding prefix for such constants as `LOCAL_STORAGE_` or `LOCAL_STORAGE_KEY_`. It would simpify reading code.
```suggestion
import { CHAT_ID_STORE_KEY, CHAT_MESSAGES_STORE_KEY } from 'src/constants';
```

Reply:
Good point, makes sense 👍